ROOTFS/usr/bin/waggle-rpi-watchdog

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+import subprocess
+import time
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+last_connection_time = 0
+
+def update_systemd_watchdog():
+    try:
+        subprocess.check_call(["systemd-notify", "WATCHDOG=1"])
+    except subprocess.CalledProcessError:
+        logging.warning("skipping reset of systemd watchdog")
+
+
+def time_now():
+    return time.monotonic()
+
+
+def seconds_since(start):
+    return time.monotonic() - start
+
+def nx_connection_ok():
+    try:
+        return (
```

Comment:
maybe we should add more tests... pinging the RPi is one thing, but what about the RPi's ability to ping the NX?  what else do we need to test to ensure the RPi is healthy and working